app-modules/notification/src/Notifications/Channels/MailChannel.php

```diff
@@ -90,7 +89,7 @@ public function send($notifiable, Notification $notification): void
             'content' => $message->toArray(),
             'recipient_id' => $recipientId,
             'recipient_type' => $recipientType,
-            ...(RoutedEngagements::active() ? ['recipient_address' => is_array($recipientAddress) ? null : $recipientAddress] : []),
+            ...(['recipient_address' => is_array($recipientAddress) ? null : $recipientAddress]),
```

Comment:
You can get rid of the array expansion here. That was just being used to inline the feature flag check.